Makefile

```
@@ -135,6 +152,16 @@ test-integration: manifests generate fmt vet envtest ginkgo ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" \
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
+USE_EXISTING_KIND ?= false
```

It might not be a kind cluster

```suggestion
USE_EXISTING_CLUSTER ?= false
```

---

Done.